### ISSUE TITLE
Depth buffer access for post-processing software like Reshade

### ISSUE THREAD

**Issue by Margen67** [Cxbx-Reloaded/Cxbx-Reloaded#2162]
## Quick summary
Cxbx-Reloaded no longer provides a way of accessing the depth buffer after #1667 was merged.

## Details
Post-processing software like Reshade require access to the depth buffer for effects like Ambient Occlusion, Depth of Field, etc.

### System Configuration:
N/A

### Additional Information (if any):
Related: #1598

**Comment by PatrickvL** [comment]
Allocating and populating depth buffer would only be beneficial for 3rd party tools, Cxbx-Reloaded won't benefit from it (and would suffer slight performance overhead).
Actually, it's not yet entirely clear IF we'd be able to access Xbox depth buffer and how to copy it's content to a host counterpart. This may involve more work than may be anticipated upfront.

So, I guess the core dev team won't do much on this, but we probably would consider a pull request that adds this, but only if it's accompanied by a feature toggle (default off).
